src/utils/dates.ts

```diff
@@ -19,3 +19,14 @@ export function moreThan24HoursPassed(purchaseDate: Date): boolean {
 
 	return timeElapsedInMilliseconds >= ONE_DAY_IN_MILLISECONDS;
 }
+
+export function getDaysBetweenDates(date1: Date, date2: Date): number {
+	const numberOfDaysBetween =
+		Math.abs(date1.getTime() - date2.getTime()) / ONE_DAY_IN_MILLISECONDS;
```

Comment:
totally valid, consider Math.floor so you're not having to round your return value